### ISSUE TITLE
CTRL+A doesn't "highlight all" when typing in a text field

### ISSUE THREAD

**Issue by AegarPyke** [okamstudio/godot#1094]
For example, if I'm creating a new script for a current node, a dialog pops up where I need to type out various path names and such. If I make a mistake, I typically use CTRL+A to highlight all of the text so I can start over. In Godot, however, the CTRL+A shortcut doesn't have any kind of text-highlighting functionality. Instead, it's bound to "Add New Node" and causes the new node dialog to show up.

**Comment by akien-mga** [comment]
I agree that it's a bit confusing, I've had to learn not to use Ctrl+A at all in Godot indeed.

@godotengine/user-interface @reduz What do you think about this? Should text selection with Ctrl+A be implemented and the hotkey for "Add New Node" set to something else?

**Comment by MarianoGnu** [comment]
About this, i haven't talked to anyone, but i was thinking into implementing a global setup for Editor, so you can setup a "Select All" shorcut and it should work in all contexts, like shader editor, visual shader editor, canvas item editor, and souch. Same for "Duplicate" "Copy" "Paste" etc.
Right now this idea has no priority because most shorcuts are working, so, this issue must e handled before.
Adding the shorcut to GdScript editor and accept event should prevent propagating to canvas item editor, so oth shourcuts may exist at the same time.

**Comment by MarianoGnu** [comment]
@akien-mga i just confirmed this issue is fixed, Ctrl+A selects all in script editor and shader editor (may be common for RichTextEditor), didn't work when renaming a node thought.

**Comment by akien-mga** [comment]
See also #2745 for a general-purpose feature request about hotkey listing and redefinition.

**Comment by neikeq** [comment]
If I am not wrong, it shouldn't be necessary to change the hotkey for "Add New Node". It would be enough to make LineEdit capture it, like TextEdit already seems to do.
